frontend/src/reducer/root.js

```diff
@@ -31,7 +31,10 @@ const rootReducer = combineReducers({
   contentType,
   timezone,
   system,
-  searchResult,
+  simpleSearch: searchResult('simple'),
+  contentSearch: searchResult('content'),
+  userSearch: searchResult('user'),
+  spaceSearch: searchResult('space'),
```

Comment:
I would expect these strings to come from constant.
You already have a ADVANCED_SEARCH_TYPE constant for the advanced ones. Maybe you need a constant for each 4 types of searches

Reply:
will fix, there is the SEARCH_TYPE too

Reply:
ok, then, if you keep both constants; I think ADVANCED_SEARCH_TYPE should be constructed from SEARCH_TYPE